app/server/utils/metrics/transfer_stats.py

```diff
@@ -70,15 +70,6 @@ def __init__(self, timeseries_unit = 'day'):
             filterable_by=self.filterable_attributes,
             bypass_user_filters=True))
 
-        transfer_use_breakdown_query = db.session.query(CreditTransfer.transfer_use.cast(JSONB),func.count(CreditTransfer.transfer_use)).group_by(CreditTransfer.transfer_use.cast(JSONB))
```

Comment:
Killing this stat since this case will be covered by group_by (no need to keep the legacy one in this case)